Bots/DotNet/Consumers/CodeFirst/SimpleHostBot-2.1/appsettings.json

```
@@ -5,9 +5,34 @@
   "SkillHostEndpoint": "http://localhost:35005/api/skills/",
   "BotFrameworkSkills": [
     {
-      "Id": "EchoSkillBot",
+      "Id": "EchoSkillBotDotnet",
+      "AppId": "",
+      "SkillEndpoint": "http://localhost:35400/api/messages"
+    },
+    {
+      "Id": "EchoSkillBot21Dotnet",
       "AppId": "",
       "SkillEndpoint": "http://localhost:35405/api/messages"
+    },
+    {
+      "Id": "EchoSkillBotV3Dotnet",
+      "AppId": "",
+      "SkillEndpoint": "http://localhost:35407/api/messages"
+    },
+    {
+      "Id": "EchoSkillBotJS",
+      "AppId": "",
+      "SkillEndpoint": "http://localhost:39783/api/messages"
```

Update port to the one in the infra diagram
```suggestion
      "SkillEndpoint": "http://localhost:36400/api/messages"
```